docker/relay/cf-execd.Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM centos:8
+
+RUN \
+        ln -sf /bin/true /usr/sbin/service && \
+        ln -sf /bin/true /bin/systemctl && \
+        ln -sf /bin/true /usr/bin/systemctl
+
+RUN \
+        yum -y install diffutils curl && \
+        curl -o rudder-setup https://repository.rudder.io/tools/rudder-setup && \
+        sh rudder-setup add-repository 6.1 && \
+        yum -y install rudder-agent && \
+        yum -y install rudder-server-relay
+
+RUN \
+        mkdir  /data /apache_conf_file && \
+        ln -sf /data/uuid.hive /opt/rudder/etc/uuid.hive && \
+        ln -sf /data/ssl/agent.cert /opt/rudder/etc/ssl/agent.cert && \
+        ln -sf /data/ppkeys/localhost.pub /var/rudder/cfengine-community/ppkeys/localhost.pub && \
+        ln -sf /data/ppkeys/localhost.priv /var/rudder/cfengine-community/ppkeys/localhost.priv
+COPY \
+        script.sh .
+
+#Make symbolic links to mount volume from cf-execd to httpd
+#To make the files under /apache_conf_file act like origin files, we make a copy of the origin file to /apache_cong_file
+RUN \
+        sh script.sh && \
```

Comment:
why using a script for cp but not ln?

Reply:
ln didn't work even on the cf-execd container so i thought about making a copy of the files to the directory that we want to share and make then a symbolic link